<?xml version="1.0"?>

<launch>
       

       <node pkg="mm_mobile_control" type="mm_trans.py" name="mm_trans" output="screen"/>  
 
       <node pkg="mm_arm_control" type="moveit_ik_test.py" name="mm_arm" output="screen"/>  

       <node pkg="mm_visual_postion" type="calcuPose" name="mm_visual_position" output="screen"/> 
	
       <node pkg="mm_endmotor_control" type="serial_port" name="mm_endmotor" output="screen"/>  
      
       <node pkg="mm_robot_decision" type="mm_decison_c" name="mm_decision" output="screen" />

	<node pkg="mm_visual_postion" type="light" name="light11" output="screen" />

</launch>
